usr/src/uts/common/os/log_sysevent.c

```diff
@@ -641,6 +642,10 @@ sysevent_add_attr(sysevent_attr_list_t **ev_attr_list, char *name,
 		error = nvlist_add_hrtime(*ev_attr_list, name,
 		    se_value->value.sv_time);
 		break;
+	case SE_DATA_TYPE_NVLIST:
```

Comment:
Interesting.  I know that having an `nvlist` inside an `nvlist` is completely valid, but I don't know how to rectify any possible issue with FreeBSD - any thoughts from others on this would be appreciated.